src/modules/people-list/people-list-item.tsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import Swipeable from 'react-native-swipeable';
+import {IProps} from './people-list-item.type';
+import Styled from 'styled-components/native';
+
+import {minorScale} from '../scales';
+import Text from '../text/text';
+import Avatar from '../avatar/avatar';
+import Checkbox from '../checkbox/checkbox';
+import Heading from '../heading/heading';
+
+// Wrapper
+const StyleWrapper = Styled.View`
```

Comment:
This should be StyledWrapper (notice the "d" in "styled"). Same thing for the other components.

Reply:
I have changed